pkg/system/config.go

```diff
@@ -39,22 +40,19 @@ func InitConfig() error {
 		return fmt.Errorf("failed to init config dir: %w", err)
 	}
 
-	configFile, err := ensureConfigFile(configDir)
-	if err != nil {
-		return fmt.Errorf("failed to init config file: %w", err)
-	}
-
 	userIDKey, err := ensureUserIDKey(configDir)
 	if err != nil {
 		return fmt.Errorf("failed to init user ID key file: %w", err)
 	}
 	viper.SetDefault("user-id-key", userIDKey) // rsa key for identifying user
 
 	// Settings and initialisation for viper:
-	viper.SetConfigFile(configFile) // provided or created config file
-	viper.SetConfigType("yaml")     // config is always a yaml file
-	viper.AutomaticEnv()            // try to read config from env if possible
-	viper.SetEnvPrefix("bacalhau")  // BACALHAU_<key> is encoding for env vars
+	viper.SetConfigName("config")
+	viper.SetConfigType("toml")
+	//viper.SetConfigType("yaml")
```

Comment:
Why TOML when our other stuff uses YAML?

Reply:
For development purposes I made it TOML; personally, I find it easier to read. But Viper will handle this for us: it supports JSON, TOML, YAML, HCL, and others. I'll be sure to make the final change default to YAML.